scripts/performance/memory-performance/compute-memory-performance.js

```diff
@@ -0,0 +1,115 @@
+const puppeteer = require('puppeteer')
+const { timeout } = require('../../lib/execution-utils')
+const { fetchPR, LOCAL_BRANCH } = require('../../lib/git-utils')
+const NUMBER_OF_RUNS = 40 // Rule of thumb: 30 runs should be enough to get a good average
+const TASK_DURATION = 1000
+const ACTION_NAMES = [
+  'RUM - add global context',
+  'RUM - add action',
+  'RUM - add error',
+  'RUM - add timing',
+  'RUM - start view',
+  'RUM - start/stop session replay recording',
+  'Logs - log message',
+]
+
+async function computeMemoryPerformance() {
+  const results = []
+  const pr = await fetchPR(LOCAL_BRANCH)
+  const bundleUrl = pr
+    ? `https://www.datad0g-browser-agent.com/pull-request/${pr.number}/datadog-rum.js`
+    : 'https://www.datadoghq-browser-agent.com/datadog-rum-canary.js'
+  const benchmarkUrl = pr
+    ? `https://datadoghq.dev/browser-sdk-test-playground/performance/?prNumber=${pr.number}`
+    : 'https://datadoghq.dev/browser-sdk-test-playground/performance/'
+  for (let i = 0; i < ACTION_NAMES.length; i++) {
+    const sdkTask = ACTION_NAMES[i]
```

Comment:
**Suggestion:** you should find a name for those tasks and stick with it. Here, you use "sdk task", "action", "test" (in `runTest`), so it is pretty confusing.